### DISCUSSION TITLE
Quotes in string from encodeToString()

### DISCUSSION THREAD

**Discussion by medvedy** [charleskorn/kaml#501] [Q&A] [answered]
Hello,

the example under "Serializing from a Kotlin object to YAML" prints the following:

leader: "Amy"
members:
- "Bob"
- "Cindy"
- "Dan"

Is there a built-in possibility to print it without the quotes? Normally, the values in YAML should not have quotes.

Thanks for your help!

**Comment by charleskorn** [comment, marked as answer]
Yep, try setting `YamlConfiguration.singleLineStringStyle` and `YamlConfiguration.multiLineStringStyle` to `Plain`.

**Comment by medvedy** [comment]
Thanks, works!
